### PR TITLE
Change context text for pull-kubernetes-cross

### PR DESCRIPTION
**Pull request by cblecker**
Noticed that the context for this test is different than the rest.

### CODE REVIEW

**Comment by spxtr** [comment]
cross test or cross build?

**Comment by cblecker** [comment]
Ah, yeah it does look like a build: https://github.com/kubernetes/test-infra/blob/49d372460871d6c34e3510f80b6cc083d86d6e5f/jobs/config.json#L295-L302

Will change.

File: prow/config.yaml [hunk 1]
```diff
@@ -121,9 +121,9 @@ presubmits:
             path: /mnt/disks/ssd0
 
   - name: pull-kubernetes-cross
-    context: pull-kubernetes-cross
-    rerun_command: "@k8s-bot pull-kubernetes-cross test this"
-    trigger: "@k8s-bot pull-kubernetes-cross test this"
+    context: Jenkins cross build
+    rerun_command: "@k8s-bot cross test this"
```
Comment (spxtr) [inline prow/config.yaml]: It's a bit of a bicycle shed, but maybe this should also be `@k8s-bot cross build this`? I think that's what it was in the past, but it's different now.

Comment (cblecker) [inline prow/config.yaml]: I don't mind changing it, but for consistency, here's what the bazel build trigger is: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L45-L50

Is it still a good idea to change it?

Comment (ixdy) [inline prow/config.yaml]: I think it was "test" because we may have wanted to make it a required status at one point?

**Comment by fejta** [comment]
Please hold off on merging this. I do not like the context change.

**Comment by fejta** [comment]
The actual intent here is to switch all the contexts to use the job name.

Specifically I want to move to a world where if the job is pull-kubernetes-e2e-gce that:
1) The context for the github check mark is pull-kubernetes-e2e-gce
2) I can assume an `@k8s-bot pull-kubernetes-e2e-gce test this` comment triggers this test.

I migrated everything that does not always run to match this pattern. Blocking jobs are more complicated because I am not sure the best way to migrate the context for 700 open PRs.

What is the need for this change?

**Comment by cblecker** [comment]
The intent of this PR was consistency, as this job's context and command didn't match the others. If we're switching to a different system, then this change isn't likely needed.. however it's confusing having some one way, and some the other.

**Comment by fejta** [comment]
Agreed, I want to shift everything to the next context... @spxtr how can we migrate contexts? I would like to do this.

**Comment by spxtr** [comment]
We can, the only catch is the fact that the submit queue uses contexts for deciding what to merge, so we need to update that at the same time.

/lgtm cancel

**Comment by cblecker** [comment]
Closing this PR as it's not needed :)

**Pull request closed without merging**